prisma/schema.prisma

```diff
@@ -12,17 +12,18 @@ generator dbml {
 }
 
 model Wallet {
-  address     String             @id
-  label       String             @default("")
-  avatar      String             @default("")
-  createdAt   DateTime           @default(now())
-  lastLogin   DateTime?
-  nonce       String             @unique @default(uuid())
-  role        Role               @default(User)
-  creator     Creator?
-  comics      WalletComic[]
-  comicIssues WalletComicIssue[]
-  newsletter  Newsletter?
+  address       String             @id
+  label         String             @default("")
+  avatar        String             @default("")
+  createdAt     DateTime           @default(now())
+  lastLogin     DateTime?
+  nonce         String             @unique @default(uuid())
+  role          Role               @default(User)
+  creator       Creator?
+  comics        WalletComic[]
+  comicIssues   WalletComicIssue[]
+  newsletter    Newsletter?
+  WalletCreator WalletCreator[]
```

Comment:
1st comment: `WalletCreator` field here should be lowercase. I assume that prisma VSC extension automatically generated this field for you and used capital case

2nd comment: let's give this a more meaningful name than `WalletCreator`. Maybe something like `followedCreators`?

Reply:
Yes prisma autogenerate did this. I will make a change